web/components/login/Login.js

```diff
@@ -75,7 +71,7 @@ class Login extends React.Component {
     );
   }
 
-  signedInView() {
+  signedInView = () => {
```

Comment:
The view functions don't need to be arrow functions right?

Reply:
well this one access `this.props` 🤷‍♂️

Reply:
Yes? this.props works just fine since signedInView is called via this.signedInView().

() => {} explicity binds this to the function. This is mostly a style issue really

Reply:
There's some performance penalties associated with using foo = () => {} over foo(){} since the first doesn't end up in the prototype and is an object property instead.

https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1

Seems like @autobind is the most performant way to do this overall. Though, the performance difference is probably negligible anyways.

Reply:
oops yeah you're right about the view functions, I've fixed that 👍 thanks!

> Though, the performance difference is probably negligible anyways.

💭 

> We should forget about small efficiencies, say about 97% of the time: premature optimization is the root of all evil.